exercises/rna-transcription/src/RnaTranscription.idr

```diff
@@ -0,0 +1,17 @@
+module RnaTranscription
+
+import Data.Vect
+
+%access export
+%default total
+
+namespace DnaStrand
```

Comment:
Why did you create separate namespaces here? Are they necessary?

Reply:
Because otherwise we get already defined errors since the constructors of RnaStrand and DnaStrand share the same names.

Reply:
Ah, that makes perfect sense. Thanks.